### PR TITLE
Adding note about oc run

### PR DESCRIPTION
**Pull request by bergerhoffer**
Release notes mention related to #23291.

Preview (internal): http://file.rdu.redhat.com/~ahoffer/2020/ocp-4-5-release-notes.html#ocp-4-5-oc-run-pod

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@soltysh Can you please review?

Preview: http://file.rdu.redhat.com/~ahoffer/2020/ocp-4-5-release-notes.html#ocp-4-5-oc-run-pod

**Comment by bergerhoffer** [comment]
@zhouying7780 Can you please review this release notes update? Thanks!

Preview: http://file.rdu.redhat.com/~ahoffer/2020/ocp-4-5-release-notes.html#ocp-4-5-oc-run-pod

File: release_notes/ocp-4-5-release-notes.adoc [hunk 1]
```diff
@@ -368,6 +368,11 @@ deprecated in {product-title} 3.11 and has been removed. You must use the
 deprecated in {product-title} 3.11 and have been removed. You must use the
 `--suggest` flag with this command instead.
 
+[id="ocp-4-5-oc-run-pod"]
+==== The `oc run` OpenShift CLI command now only creates Pods
+
+The `oc run` command can now only be used to creates Pods. Use the `oc create` command instead to create other resources.
```
Comment (codyhoag) [inline release_notes/ocp-4-5-release-notes.adoc:374]: s/creates Pods/create Pods

**Review by soltysh — APPROVED** [review]
lgtm

**Review by codyhoag — COMMENTED** [review]
One suggestion. LGTM otherwise!

File: release_notes/ocp-4-5-release-notes.adoc (hunk 1 above)
Comment (bergerhoffer) [reply release_notes/ocp-4-5-release-notes.adoc:374]: Good catch, thanks!

**Pull request merged**